interfaces/HO-Portal/src/app/iframe/recipient.page.spec.ts

```diff
@@ -9,7 +13,13 @@ describe('Recipient.PageComponent', () => {
   beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
       declarations: [RecipientPage],
-      imports: [IonicModule.forRoot()],
+      imports: [
+        IonicModule.forRoot(),
+        RouterTestingModule,
+        TranslateModule.forRoot(),
+        HttpClientModule,
```

Comment:
```suggestion
        HttpClientTestingModule,
```
I haven't checked the exact name, but the point is that the tests shouldn't make REAL HTTP-requests, but fake/mocked ones only.